master/usage/decommissioning-a-node.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Decommissioning a Calico Node
+---
+
+With each Calico node there is a [Node resource][Node resource reference]
```

Comment:
I think this should be less focused on explaining what a Node resource is and more focused on explaining why you might want to follow the instructions on this page.